command/src/state.rs

```diff
@@ -52,10 +50,9 @@ pub struct ConfigState {
     pub http_listeners: HashMap<SocketAddr, (HttpListenerConfig, bool)>,
     pub https_listeners: HashMap<SocketAddr, (HttpsListenerConfig, bool)>,
     pub tcp_listeners: HashMap<SocketAddr, (TcpListenerConfig, bool)>,
-    /// indexed by (address, hostname, path)
-    pub http_fronts: BTreeMap<RouteKey, HttpFrontend>,
-    /// indexed by (address, hostname, path)
-    pub https_fronts: BTreeMap<RouteKey, HttpFrontend>,
+    /// HTTP and HTTPS frontends, indexed by a summary of each front's address;hostname;path, for uniqueness.
```

Comment:
How do you discriminate frontends with tls from whose that they do not have tls enabled?

Reply:
- #909